Update azurerm_eventhub_namespace with optional dedicated_cluster_id (#7548)
terraform-providers/terraform-provider-azurerm: website/docs/d/eventhub_namespace.html.markdown

* `zone_redundant` - Is this EventHub Namespace deployed across Availability Zones?

> * `dedicated_cluster_id` - Is this EventHub Namespace deployed on a Dedicated Event Hubs Cluster?

Reviewer: ```suggestion
* `dedicated_cluster_id` - The ID of the EventHub Dedicated Cluster where this Namespace exists.
```
